code/data_structures/Leftist_Tree/LeftistPriorityQueue.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <algorithm>
+
+struct Leftist {
```

Comment:
Could you add the `insert` function using the `merge` function? so that the code is complete.

This will make this good to be merge 👍